src/activities/ballcatch/Ballcatch.qml

```diff
@@ -209,23 +213,25 @@ ActivityBase {
             id: rightShift
             mirror: true
             x: background.width - background.width / 4
-            y: rightHand.y - height / 2
+            y: background.isVertical ? rightHand.y - height : rightHand.y - height / 2
             source: "qrc:/gcompris/src/activities/ballcatch/resource/arrow_key.svg"
+            scale: background.isVertical ? 0.75 : 1.0
+            smooth: true
             opacity: items.rightPressed ? 1 : 0.5
             visible: !ApplicationInfo.isMobile
         }
 
         // Instructions
         BorderImage {
```

Comment:
The text hides Tux's head, maybe it would be better to use IntroMessage component to display the instruction?

Reply:
It does not on mine, a screenshot will help and i'll checkout how IntroMessage component works out on this.

Reply:
Here is how it looks https://pasteboard.co/H4rAY1Q.png

Reply:
Sticking with the older intro message is better.

Reply:
It will be really helpful if there were a screenshot. i suppose in other languages (except English) this is creating a problem.

Reply:
https://pasteboard.co/H508eX4.png

Reply:
I've added the IntroMessage Component as you suggested.